packages/luna/src/components/Chat/ChatMessageGroup/ChatMessageGroup.tsx

```diff
@@ -44,7 +45,7 @@ export const ChatMessageGroup: React.FC<{
       <div className="tw-w-14 tw-h-14 tw-overflow-hidden tw-rounded-full tw-flex-shrink-0">
         <Avatar
           alt={orUndefined(name)}
-          src={orUndefined(image)}
+          src={orUndefined(asFullAssetUrl(image || ""))}
```

Comment:
This is not correct. 
```suggestion
          src={image ? asFullAssetUrl(image): undefined}
```